JustBrewIt/Pages/Search.cshtml

```diff
@@ -1,35 +1,34 @@
 ﻿@page
 @model JustBrewIt.Pages.SearchModel
 @{
+
+    ViewData["Title"] = "Search Page";
     var BreweryAPI = (JustBrewItQuickType.JustBrewIT[])ViewData["BreweryAPI"];
+
+
 }
 
 <form action="" method="POST" style="margin-top: 50px;">
-    <div class="form-group">
-        <label class="control-label" for="BreweryCity">Search Brewries by City</label>
-        <input type="text" class="form-control" name="BreweryCity" id="BreweryCity" />
-    </div>
-
-    <div class="form-group">
-        <label class="control-label" for="BreweryType">Search Brewries by Type</label>
-        <select class="form-control" name="BreweryType" id="BreweryType">
-            <option value=""></option>
-            <option value="micro">Micro</option>
-            <option value="nano">Nano</option>
-            <option value="regional">Regional</option>
-            <option value="brewpub">Brewpub</option>
-            <option value="large">Large</option>
-            <option value="planning">Planning</option>
-            <option value="bar">Bar</option>
-            <option value="contract">Contract</option>
-            <option value="proprietor">Proprietor</option>
-            <option value="closed">Closed</option>
-        </select>
-    </div>
-    <div class="form-group">
-        <input type="submit" value="Search" class="btn btn-sm btn-primary" />
-    </div>
+    <label width="30" class="control-label" for="BreweryCity">Search Brewries by City</label>
+    <input type="text" placeholder="Enter city name" name="BreweryCity" id="BreweryCity" />
+
+    <label class="control-label" for="BreweryType">Search Brewries by Type</label>
+    <select class="featurette-heading"  name="BreweryType" id="BreweryType">
+        <option value="">Select type</option>
+        <option value="micro">Micro</option>
+        <option value="nano">Nano</option>
+        <option value="regional">Regional</option>
+        <option value="brewpub">Brewpub</option>
+        <option value="large">Large</option>
+        <option value="planning">Planning</option>
+        <option value="bar">Bar</option>
+        <option value="contract">Contract</option>
+        <option value="proprietor">Proprietor</option>
+        <option value="closed">Closed</option>
+    </select> &nbsp;&nbsp;
+    <input type="submit" value="Search"  class="btn btn-sm btn-primary" />
```

Comment:
Suggestion 2:  Search input boxes look better in one line.
Fixed:  modified alignment by removing extra <div>  and added & nbsp;&nbsp; space between search box and submit button.
https://www.jotform.com/blog/how-to-make-a-web-design-look-good/